edge/pkg/devicetwin/dtcommon/util.go

```diff
@@ -90,6 +93,68 @@ func ConvertDevice(device *v1alpha2.Device) (*pb.Device, error) {
 		klog.Errorf("fail to unmarshal device %s with err: %v", device.Name, err)
 		return nil, err
 	}
+	if device.Spec.Protocol.CustomizedProtocol != nil {
+		// interface data to anypb.Any data
+		configAnyData := make(map[string]*anypb.Any)
+		for k, v := range device.Spec.Protocol.CustomizedProtocol.ConfigData.Data {
+			anyValue, err := DataToAny(v)
+			if err != nil {
+				return nil, err
+			}
+			configAnyData[k] = anyValue
+		}
+		edgeDevice.Spec.Protocol.CustomizedProtocol.ConfigData.Data = configAnyData
+	}
+	if device.Spec.Protocol.Common.CustomizedValues != nil {
+		// interface data to anypb.Any data
+		configAnyData := make(map[string]*anypb.Any)
+		for k, v := range device.Spec.Protocol.Common.CustomizedValues.Data {
+			anyValue, err := DataToAny(v)
+			if err != nil {
+				return nil, err
+			}
+			configAnyData[k] = anyValue
+		}
+		edgeDevice.Spec.Protocol.Common.CustomizedValues.Data = configAnyData
+	}
+	var edgePropertyVisitors []*pb.DevicePropertyVisitor
+	for i := range device.Spec.PropertyVisitors {
+		var item *pb.DevicePropertyVisitor = new(pb.DevicePropertyVisitor)
```

Comment:
type can be omitted, `var item = new(pb.DevicePropertyVisitor)`

Reply:
Done.